src/shell/commands/duplication.cpp

```diff
@@ -87,6 +84,14 @@ bool add_dup(command_executor *e, shell_context *sc, arguments args)
     uint32_t remote_replica_count = 0;
     PARSE_OPT_UINT(remote_replica_count, 0, {"-r", "--remote_replica_count"});
 
+    fmt::print("trying to add duplication [app_name: {}, remote_cluster_name: {}, "
```

Comment:
We can use fmt::println to avoid adding `\n` manually.

Other places are the same.